pkg/operator/cluster/ceph/osd/osd.go

```diff
@@ -189,13 +204,26 @@ func (c *Cluster) Start() error {
 				}
 			}
 		} else {
-			logger.Infof("osd replica set started for node %s", n.Name)
+			logger.Infof("osd pod started for node %s", n.Name)
 		}
 
 		// wait for the current node's orchestration to be completed
-		if err := c.waitForCompletion(n.Name); err != nil {
+		if status, err := c.waitForCompletion(n.Name); err != nil {
```

Comment:
This `waitForCompletion` is done for adding new nodes to the cluster, which shouldn't be needed for a new deployment. I believe this only is needed for allowing time for existing PGs to adjust when adding nodes to an existing cluster.